libc/src/lib.rs

```diff
@@ -1,48 +0,0 @@
-#![warn(missing_docs)]
-#![cfg_attr(feature = "strict", deny(warnings))]
-#![no_std]
-
-//! libc crate
-
-#[macro_use]
-extern crate cfg_if;
-
-cfg_if! {
-	if #[cfg(feature="ext_memops")] {
-		extern "C" {
-			fn ext_memcmp(cx: *const u8, ct: *const u8, n: usize) -> i32;
-			fn ext_memcpy(dest: *mut u8, src: *const u8, n: usize) -> *mut u8;
-			fn ext_memmove(dest: *mut u8, src: *const u8, n: usize) -> *mut u8;
-			fn ext_memset(dest: *mut u8, c: i32, n: usize) -> *mut u8;
-		}
-
-		// Declaring these function here prevents Emscripten from including it's own verisons
-		// into final binary.
-
-		/// memcmp extern
-		#[no_mangle]
-		pub unsafe extern "C" fn memcmp(cx: *const u8, ct: *const u8, n: usize) -> i32 {
-			ext_memcmp(cx, ct, n)
-		}
-
-		/// memcpy extern
-		#[no_mangle]
-		pub unsafe extern "C" fn memcpy(dest: *mut u8, src: *const u8, n: usize) -> *mut u8 {
-			ext_memcpy(dest, src, n)
-		}
-
-		/// memmove extern
-		#[no_mangle]
-		pub unsafe extern "C" fn memmove(dest: *mut u8, src: *const u8, n: usize) -> *mut u8 {
-			ext_memmove(dest, src, n)
-		}
-
-		/// memset extern
-		#[no_mangle]
-		pub unsafe extern "C" fn memset(dest: *mut u8, c: i32, n: usize) -> *mut u8 {
-			ext_memset(dest, c, n)
-		}
-	} else {
-		extern crate rlibc;
```

Comment:
I think only this line should be removed, we'd better keep external memory ops for libc

Reply:
I tried it locally and that single-line change alone fixes the link issue as well.
Should I make a new PR, or commit a revert?

Reply:
Oh, you can update this PR since it's not merged :)

Reply:
Makes sense. 
I think it would make sense to move this code into the top-level crate also (as a separate mod, but not a crate). I don't see any reason in the current granularity. Have a separated `pwasm-libc` crate for the purpose it serves currently has not too much sense to me.

Reply:
Ah, yeah, it is a special feature, then we can probably comment this line indeed.

However, I'm having a hard time to understand/remember why would we need this feature in the first place. Substrate doesn't use these, nor these `ext_*` functions are provided by the pwasm runtime.

Reply:
Change committed and passed the tests!